projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -55,17 +56,17 @@ const BarChartInternal: FC< BarChartProps > = ( {
 	className,
 	margin,
 	withTooltips = false,
-	showLegend = false,
-	legendOrientation = 'horizontal',
-	legendAlignmentHorizontal = 'center',
-	legendAlignmentVertical = 'bottom',
-	legendShape = 'rect',
+	showLegend = BAR_CHART_DEFAULTS.showLegend,
```

Comment:
I'm not a big fan of centralizing the defaults here as it would make the defaults less obvious. I understand the rational behind this, but I don't really see much benefit doing this than caveats, so would still lean to keeping them as before 🤔

Reply:
Ok interesting take, thanks for your opinion here @kangzj. I like this approach (obviously, as I am the one proposing it 😂 ) but I am not married to the idea and I'm open to dropping the default centralization and just shipping the styling fixes if that's what the majority prefer. Also keen to hear @adamwoodnz  opinion too.

Reply:
I tend to agree - I'm not sure the abstraction has much benefit beyond sharing legend defaults, if they are consistent between chart types. I feel like the rest of the defaults are different enough that having them co-located with the component makes more sense. Did the idea stem from creating the shared legend config?

Maybe the legend defaults can be exported from that index file and charts can import them?